distribution/client/src/lib/update.ts

```diff
@@ -163,57 +198,76 @@ export default class Update {
        while it's not been yet marked as tainted in the backend?
      * how to report that borked case in a clear way
   */
-  restartJenkins(rollingBack?: boolean) { // Add param to stop recursion?
-    Supervisord.restartProcess('jenkins');
+  restartJenkins() {
 
-    const messageWhileRestarting = 'Jenkins should now be online, health checking!';
+    const jenkinsIsRestarting = Supervisord.restartProcess('jenkins');
+
+    const messageWhileRestarting = 'Jenkins is being restarted, health checking!';
     UI.publish(messageWhileRestarting);
     logger.info(messageWhileRestarting);
```

Comment:
I believe I missed this in previous code reviews, similar to the prior comments, these two lines can be collapsed into one call.